sample_project/Assets/SampleViewer/Samples/Viewshed/Scripts/ViewshedCamera.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2025 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+//using System.Numerics;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+[ExecuteAlways]
```

Comment:
Just curious, but whats the logic behind having this always executed? If this is to behave similar to other samples then I feel like it shouldn't need to always be executed. But that could just be me. I also haven't tested it as of writing this.